### ISSUE TITLE
Issue with `strs` mapper

### ISSUE THREAD

**Issue by sardaukar** [maiha/toml-config.cr#1]
This 

```crystal
module Brucite
  class ThemeConfig < TOML::Config
    str "name", name
    str "license", license
    str "license_link", license_link
    str "description", description
    str "homepage", homepage
    str "min_version", min_version
    strs "layouts", layouts

    as_hash "author"
    as_hash "ported_by"
  end
end
```

gives me 

```
Showing last frame. Use --error-trace for full trace.

In src/theme_config.cr:9:21

 9 | strs "layouts", layouts
                     ^------
Error: undefined local variable or method 'layouts' for Brucite::ThemeConfig.class
```

but 

```crystal
module Brucite
  class ThemeConfig < TOML::Config
    str "name", name
    str "license", license
    str "license_link", license_link
    str "description", description
    str "homepage", homepage
    str "min_version", min_version

    as_hash "author"
    as_hash "ported_by"

    def layouts
      strs("layouts")
    end
  end
end
```
works. What am I doing wrong on the first example?

**Comment by maiha** [comment]
`str` has both a **class-level macro** and an **instance method**.
On the other hand, `strs` has only an **instance method**.
In short, the `strs` **macro** is undefined. I'll fix that in a minute. 😃

**Comment by maiha** [comment]
@sardaukar released 0.6.0
```crystal
  strs "layouts"
```
now should work!

##### FYI
There are some breaking changes.  The `int` and `float` APIs have been removed and replaced by APIs with specific types such as `i32`, `i64`, `f32`, and `f64`, respectively.

**Comment by sardaukar** [comment]
Thanks!
